lldb/tools/lldb-dap/LLDBUtils.cpp

```diff
@@ -162,4 +162,13 @@ GetEnvironmentFromArguments(const llvm::json::Object &arguments) {
   return envs;
 }
 
+llvm::Error takeError(const lldb::SBError &error) {
```

Comment:
How about `ToError` to match `Status::ToError`? 
```suggestion
llvm::Error ToError(const lldb::SBError &error) {
```

Reply:
Done.